docs/user-guide/config.rst

```diff
@@ -54,19 +54,20 @@ This is the current default configuration::
               'v2_default_filters': {'bytes': [{'id': 'vlen-bytes'}],
                                      'numeric': None,
                                      'string': [{'id': 'vlen-utf8'}]},
-              'v3_default_codecs': {'bytes': [{'name': 'vlen-bytes'},
-                                              {'configuration': {'checksum': False,
-                                                                 'level': 0},
-                                               'name': 'zstd'}],
-                                    'numeric': [{'configuration': {'endian': 'little'},
-                                                 'name': 'bytes'},
-                                                {'configuration': {'checksum': False,
-                                                                   'level': 0},
-                                                 'name': 'zstd'}],
-                                    'string': [{'name': 'vlen-utf8'},
-                                               {'configuration': {'checksum': False,
-                                                                  'level': 0},
-                                                'name': 'zstd'}]},
+              'v3_default_compressors': {'bytes': [{'configuration': {'checksum': False,
```

Comment:
is there a simple way to ensure that this representation and the actual config don't go out of sync? not a big deal if no.

Reply:
Yes, via doctests https://github.com/zarr-developers/zarr-python/blob/feat/separate-default-codecs/docs/user-guide/config.rst?plain=1#L43